tests/src/Kernel/PatternAssertions/ListingAssertion.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\PatternAssertions;
+
+use Drupal\file\Entity\File;
+use Drupal\Tests\oe_bootstrap_theme\Kernel\Paragraphs\ParagraphsTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Class for asserting Listing paragraphs.
```

Comment:
These classes are for easy, reusable and maintainable testing or pattern rendering, so not related to paragraphs, pls reformulate